[2025-05-17T12:49:59.306+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-17T12:49:59.563+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: BinanceCryptoAPI.create_insert_sql manual__2025-05-17T12:49:53.983094+00:00 [queued]>
[2025-05-17T12:49:59.626+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: BinanceCryptoAPI.create_insert_sql manual__2025-05-17T12:49:53.983094+00:00 [queued]>
[2025-05-17T12:49:59.628+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-05-17T12:49:59.712+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): create_insert_sql> on 2025-05-17 12:49:53.983094+00:00
[2025-05-17T12:49:59.740+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=210) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-17T12:49:59.753+0000] {standard_task_runner.py:72} INFO - Started process 213 to run task
[2025-05-17T12:49:59.751+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'BinanceCryptoAPI', 'create_insert_sql', 'manual__2025-05-17T12:49:53.983094+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/ETL/BinanceETL.py', '--cfg-path', '/tmp/tmp_2mtn4gj']
[2025-05-17T12:49:59.764+0000] {standard_task_runner.py:105} INFO - Job 16: Subtask create_insert_sql
[2025-05-17T12:50:03.154+0000] {task_command.py:467} INFO - Running <TaskInstance: BinanceCryptoAPI.create_insert_sql manual__2025-05-17T12:49:53.983094+00:00 [running]> on host 42e621b39a3e
[2025-05-17T12:50:03.338+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='BinanceCryptoAPI' AIRFLOW_CTX_TASK_ID='create_insert_sql' AIRFLOW_CTX_EXECUTION_DATE='2025-05-17T12:49:53.983094+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-17T12:49:53.983094+00:00'
[2025-05-17T12:50:03.345+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-17T12:50:03.583+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-17T12:50:03.618+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-17T12:50:03.619+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=BinanceCryptoAPI, task_id=create_insert_sql, run_id=manual__2025-05-17T12:49:53.983094+00:00, execution_date=20250517T124953, start_date=20250517T124959, end_date=20250517T125003
[2025-05-17T12:50:03.688+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-17T12:50:05.216+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2025-05-17T12:50:05.264+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-17T12:50:05.279+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
